src/routes/+layout.svelte

```diff
@@ -28,6 +24,11 @@
 		if (!($settingsStore.autoCheckForUpdates === false)) await checkForUpdates();
 	});
 
+	beforeUpdate(() => {
+		loadLocale('en');
+		setLocale('en');
+	});
```

Comment:
You'll need to re-factor this for #151.
You can use my older `julia-sanfrancisco` repo [for](https://github.com/fmaclen/julia-sanfrancisco/blob/main/src/routes/%2Bpage.svelte#L41-L50) [reference](https://github.com/fmaclen/julia-sanfrancisco/blob/main/src/lib/player.ts#L62-L74).